Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12
+FROM alpine:3.13.5
```

Comment:
IMHO, it is better not to specify a patch version. Then, Docker will use the latest patch version.